xtos/include/rtos/interrupt.h

```diff
@@ -135,6 +135,11 @@ void interrupt_unregister(uint32_t irq, const void *arg);
 uint32_t interrupt_enable(uint32_t irq, void *arg);
 uint32_t interrupt_disable(uint32_t irq, void *arg);
 
+/* Zephyr compat */
+#if !defined(__ZEPHYR__)
```

Comment:
this header is under xtos/include now, can it still be included for Zephyr?

Reply:
yes for the moment - unless Zephyr native headers Kconfig is selected (but this forces out needed headers atm).